utils/cookies.ts

```diff
@@ -0,0 +1,23 @@
+import { getCookies, setCookie } from "std/http/mod.ts";
+
+export const COOKIE_NAME = "supabase_authToken";
```

Comment:
Coisas como essa é interessante ter o nome da Niivo para tambem não expor a tecnologia que usamos
```suggestion
export const COOKIE_NAME = "niivo_auth";
```